### PR TITLE
Expose cf.compact in api

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: src/rebloom.c [hunk 1]
```diff
@@ -677,6 +677,24 @@ static int CFDel_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int
     return RedisModule_ReplyWithLongLong(ctx, CuckooFilter_Delete(cf, hash));
 }
 
+static int CFCompact_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    RedisModule_AutoMemory(ctx);
+    RedisModule_ReplicateVerbatim(ctx);
```
Comment (rafie) [inline src/rebloom.c]: Move to end of function? (a-la #121)

File: src/rebloom.c [hunk 2]
```diff
@@ -677,6 +677,24 @@ static int CFDel_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int
...
+
+    if (argc != 2) {
+        return RedisModule_WrongArity(ctx);
+    }
+
+    RedisModuleKey *key = RedisModule_OpenKey(ctx, argv[1], REDISMODULE_READ | REDISMODULE_WRITE);
```
Comment (rafie) [inline src/rebloom.c]: This key comes from argv[1]. Shouldn't we check for NULL?

**Comment by rafie** [comment]
Not strictly related to this PR, but wouldn't it be a good idea to use Bloom Python client in the unit tests?

File: src/rebloom.c (hunk 2 above)
Comment (ashtul) [reply src/rebloom.c]: No, since we checked before we have two arguments

**Pull request merged**